WordPress/Classes/ViewRelated/Comments/CommentContentTableViewCell.swift

```diff
@@ -407,6 +407,7 @@ private extension CommentContentTableViewCell {
             self.likeButton.setTitle(self.likeButtonTitle, for: .normal)
             self.adjustImageAndTitleEdgeInsets(for: self.likeButton)
             self.likeButton.setTitleColor(liked ? Style.likedTintColor : Style.reactionButtonTextColor, for: .normal)
+            self.likeButton.accessibilityLabel = liked ? "\(numberOfLikes) " + .commentIsLiked : "\(numberOfLikes) " + .commentIsNotLiked
```

Comment:
WDYT of removing the trailing space from here and making it the leading space in `commentIs(Not)Liked` on L521-522 instead, so it will be no longer needed to interpolate strings here below?

```suggestion
            self.likeButton.accessibilityLabel = liked ? numberOfLikes + .commentIsLiked : numberOfLikes + .commentIsNotLiked
```

Reply:
thanks for the suggestion! that does look better. updated in https://github.com/wordpress-mobile/WordPress-iOS/pull/21420/commits/7977b2ac79012c6e70596c79bf6fe2c40c9c2560. though one thing is that since `numberOfLikes` is type `Int` had to `String`-ify it first